collector/cooling_device_linux.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !windows
+
+package collector
+
+import (
+	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs/sysfs"
+)
+
+type coolingDeviceCollector struct {
+	fs                    sysfs.FS
+	coolingDeviceCurState *prometheus.Desc
+	coolingDeviceMaxState *prometheus.Desc
+}
+
+func init() {
+	registerCollector("cooling_device", defaultEnabled, NewCoolingDeviceCollector)
```

Comment:
Let's make this string a `const coolingDevice = "cooling_device"`. This is a minor go style thing to avoid typos.